lib/executor/constant_arrival_rate.go

```diff
@@ -227,16 +229,26 @@ func (car ConstantArrivalRate) Run(ctx context.Context, out chan<- stats.SampleC
 	}
 
 	vusFmt := pb.GetFixedLengthIntFormat(maxVUs)
-	fmtStr := pb.GetFixedLengthFloatFormat(arrivalRatePerSec, 2) +
-		" iters/s, " + vusFmt + " out of " + vusFmt + " VUs active"
-
-	progresFn := func() (float64, string) {
+	progresFn := func() (float64, []string) {
 		spent := time.Since(startTime)
 		currentInitialisedVUs := atomic.LoadUint64(&initialisedVUs)
 		vusInBuffer := uint64(len(vus))
-		return math.Min(1, float64(spent)/float64(duration)), fmt.Sprintf(fmtStr,
-			arrivalRatePerSec, currentInitialisedVUs-vusInBuffer, currentInitialisedVUs,
-		)
+		progVUs := fmt.Sprintf(vusFmt+"/"+vusFmt+" VUs",
+			currentInitialisedVUs-vusInBuffer, currentInitialisedVUs)
+		progIters := fmt.Sprintf(
+			pb.GetFixedLengthFloatFormat(arrivalRatePerSec, 0)+" iters/s", arrivalRatePerSec)
+
+		right := []string{progVUs, duration.String(), progIters}
+
+		if spent > duration {
+			return 1, right
+		}
+
+		spentDuration := pb.GetFixedLengthDuration(spent, duration)
+		progDur := fmt.Sprintf("%s/%s", strings.Replace(spentDuration, "s", "", 1), duration)
```

Comment:
Can you revert the removal of `s` from the durations? I noticed it here and in a few other places, and I think we established that it didn't make sense, considering that duration values can also contain minutes and hours...

Reply:
Weird, I thought I had removed that commit, but apparently I squashed it :man_facepalming:

It's fixed in latest push.